File: JavaScript/leapYear.js
const leapYear = year => {
  if ((year % 4 === 0 && year % 100 !== 0) || year % 400 === 0) {
    console.log(`The year ${year} is a leap year.`);
  } else {
    console.log(`The year ${year} is not a leap year.`);
  }
};

leapYear(1900);

File: Python/leapYear.py
def leapYear(year):
    if((year % 4 == 0 and year % 100 !=0) or year % 400 == 0):
        print(f"The year {year} is a leap year")
    else:
        print(f"The year {year} is not a leap year")



leapYear(1900)